### ISSUE TITLE
[Feature Request] Small play / pause / next / last buttons in the taskbar display

### ISSUE THREAD

**Issue by Gladuin** [mpv-player/mpv#5226]
Hi!
I would like to see a [feature like VLC has](https://a.safe.moe/6PyD8.png), That I can pause / skip songs (Yes I use MPV as a music player) using the task bar.
This is the only feature I really miss and I am so far really liking MPV.

Thanks!

**Comment by kevmitch** [comment]
This is a [Thumbnail Toolbar](https://msdn.microsoft.com/en-us/library/windows/desktop/dd378460(v=vs.85).aspx#THUMBBARS). Not sure if this is something we want, but we do already have the progress bar.

**Comment by Dudemanguy** [comment]
Don't we have this on windows and mac these days?

**Comment by kasper93** [comment]
No, they mean buttons like those: https://learn.microsoft.com/en-us/windows/win32/shell/images/taskbar/thumbbar.png it's trivial to add, we even already use this interface for progress bar. But it needs bitmaps of the buttons and this would be annoying to store in mpv in some way. Patches welcome.
